pkg/sql/sqlstats/persistedsqlstats/flush.go

```diff
@@ -465,7 +474,7 @@ func (s *PersistedSQLStats) insertStatementStats(
 ) (rowsAffected int, err error) {
 	insertStmt := `
 INSERT INTO system.statement_statistics
-VALUES ($1, $2, $3, $4, $5, $6, $7, $8, $9, $10)
+VALUES ($1, $2, $3, $4, $5, $6, $7, $8, $9, $10, $11)
 ON CONFLICT (crdb_internal_aggregated_ts_app_name_fingerprint_id_node_id_plan_hash_transaction_fingerprint_id_shard_8,
              aggregated_ts, fingerprint_id, transaction_fingerprint_id, app_name, plan_hash, node_id)
```

Comment:
Should we also check for conflict on index recommendations here as well?
Is there a case where we want to insert into an existing stats row if our index recommendations have changed?